main/res/values/strings.xml

```diff
@@ -236,6 +236,7 @@
     <string name="info_stored_image">New image saved to:</string>
     <string name="info_storing_static_maps">Trying to store static maps</string>
     <string name="info_cache_saved">The cache has been stored locally</string>
+    <string name="err_trackable_log_not_anonymous">Logging trackables from c:geo cannot be anonymous. Please connect to the service.</string>
```

Comment:
...cannot be done anonymously.  or ...cannot be done without user name and password.

"Please connect to the service" is easier to understand when rephrased as "Please add your user name and password in the %s settings", with the service name inserted.